SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -34,62 +35,51 @@ bool WriteFlagSection(TSharedRef<TJsonWriter<>> Writer, const FString& Key, cons
 	return true;
 }
 
-bool WriteWorkerSection(TSharedRef<TJsonWriter<>> Writer, const FName& WorkerTypeName, const FWorkerTypeLaunchSection& WorkerConfig)
+bool WriteLoadbalancingSection(TSharedRef<TJsonWriter<>> Writer, const FName& WorkerType, uint32 NumEditorInstances,
```

Comment:
Ultra nit, but this was previously under WriteWorkerSection, and git is doing the diff weirdly here.

Reply:
Actually, can't be done, since this is only defined in the cpp. Nevermind.

Reply:
Could always declare the function in the cpp also